src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -119,11 +120,16 @@ export class AccountManagementService implements IAccountManagementService {
 	 * @return Promise to return an account
 	 */
 	public addAccount(providerId: string): Thenable<void> {
+		const closeAction: Action = new Action('closeAddingAccount', localize('close', "Close"), undefined, true);
```

Comment:
you might want to add something like `azure.close` for the localization tag since `close` is too common and risk prone

Reply:
I assumed anyone localizing "close" is just going to want "close"

Reply:
yes, but we don't want duplicates in the mapping that's generated
it's almost guaranteed `close` is used somewhere, and this is why we use `extension.command` everywhere as general practice